runtime panic of process inspector

Below panic was caused during my testing etcd e2e test:

```
panic: Invalid parameter n: 0

goroutine 37 [running]:
panic(0xa304a0, 0xc420fdb6c0)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/osrg/namazu/vendor/github.com/leesper/go_rng.DirichletGenerator.FlatDirichlet(0xc42048c208, 0x0, 0xc421e05b60, 0x0, 0xc421e05b60)
        /home/mitake/gopath/src/github.com/osrg/namazu/vendor/github.com/leesper/go_rng/dirichlet.go:56 +0x1ad
github.com/osrg/namazu/nmz/explorepolicy/random.(*dirichlet).dirichletSchedDeadline(0xc42048c2a8, 0x16414c0, 0x0, 0x0, 0xf4240, 0x3ff0000000000000, 0x0)
        /home/mitake/gopath/src/github.com/osrg/namazu/nmz/explorepolicy/random/dirichlet.go:56 +0x8b
github.com/osrg/namazu/nmz/explorepolicy/random.(*dirichlet).Action(0xc42048c2a8, 0xc421e5c540, 0xc4220c2050, 0xc4200d6b60, 0xc4200d6b60, 0xc4220c2050)
        /home/mitake/gopath/src/github.com/osrg/namazu/nmz/explorepolicy/random/dirichlet.go:42 +0x9c
github.com/osrg/namazu/nmz/explorepolicy/random.(*Random).makeActionForEvent(0xc42018cb40, 0x10e42e0, 0xc421e5c540, 0xc42129bf78, 0x2, 0x0, 0x0)
        /home/mitake/gopath/src/github.com/osrg/namazu/nmz/explorepolicy/random/randompolicy.go:303 +0xa7
github.com/osrg/namazu/nmz/explorepolicy/random.(*Random).dequeueEventRoutine(0xc42018cb40)
        /home/mitake/gopath/src/github.com/osrg/namazu/nmz/explorepolicy/random/randompolicy.go:323 +0x10d
created by github.com/osrg/namazu/nmz/explorepolicy/random.New
        /home/mitake/gopath/src/github.com/osrg/namazu/nmz/explorepolicy/random/randompolicy.go:118 +0x1d6
```

I used a script like below for iterating the test:
```
#! /bin/bash

for i in `seq 1 1000`; do
    GOPATH=`pwd`/gopath nmz inspectors proc -watch-interval 1ms -stdout nmz/log-$i -autopilot dirichlet.toml -cmd "./e2e.test -bin-dir bin -test.run TestCtlV3Lock"
done
```

dirichlet.toml is this:
```
explorePolicy = "random"
[explorePolicyParam]
 procPolicy = "dirichlet"
```

Any ideas? @AkihiroSuda

--- a/nmz/explorepolicy/random/dirichlet.go
+++ b/nmz/explorepolicy/random/dirichlet.go
@@ -16,6 +16,7 @@
 package random
 
 import (
+	"fmt"
 	"github.com/AkihiroSuda/go-linuxsched"
 	log "github.com/cihub/seelog"
 	"github.com/leesper/go_rng"
@@ -38,6 +39,9 @@ func (d *dirichlet) Action(event *signal.ProcSetEvent) (signal.Action, error) {
 	procs, err := parseProcSetEvent(event)
 	if err != nil {
 		return nil, err
+	}
+	if len(procs) == 0 {
+		return nil, fmt.Errorf("event %v has no process", event)
 	}
 	attrs := d.dirichletSchedDeadline(procs, time.Millisecond, 1.0)
 	for pidStr, attr := range attrs {

--- a/nmz/inspector/proc/proc.go
+++ b/nmz/inspector/proc/proc.go
@@ -70,6 +70,9 @@ func (this *ProcInspector) Serve(endCh <-chan struct{}) error {
 				log.Warn(err)
 				continue
 			}
+			if len(procs) == 0 {
+				continue
+			}
 			if err = this.onWatch(procs); err != nil {
 				log.Error(err)
 			}

